cllc-public-app/Controllers/LicensesController.cs

```diff
@@ -690,6 +785,23 @@ private MicrosoftDynamicsCRMadoxioApplication CreateApplication(string licenceId
                 application.AdoxioLicenceEstablishmentODataBind = _dynamicsClient.GetEntityURI("adoxio_establishments", adoxioLicense.AdoxioEstablishment.AdoxioEstablishmentid);
             }
 
+            // check to see if there is a related licence.
+
+            if (relatedLicenceId != null)
+            {
+                var relatedLicence = _dynamicsClient.GetLicenceByIdWithChildren(relatedLicenceId);
+                // set the establishment address to be that of the related licence.
+                application.AdoxioEstablishmentaddressstreet = relatedLicence.AdoxioEstablishment.AdoxioAddressstreet;
+                application.AdoxioEstablishmentaddresscity = relatedLicence.AdoxioEstablishment.AdoxioAddresscity;
+                application.AdoxioEstablishmentaddresspostalcode = relatedLicence.AdoxioEstablishment.AdoxioAddresspostalcode;
+
+                // TODO - the following fields do not appear to be in Dynamics yet
+
+                // Assigned Licensee == Licensee of selected licence
+                // Related Licence == Licence lookup of the manufacturers' licence
```

Comment:
Assigned Licensee is just adoxio_applicant
Created ticket to have Related Licence added (we had Related Application, which I had it confused with) https://justice.gov.bc.ca/jira/browse/LCSD-4926